### ISSUE TITLE
OHEM does Not work

### ISSUE THREAD

**Issue by gbyy422990** [WilhelmT/ClassMix#7]
hi, Its really a nice work.

ClassMix does NOT work when I changed supervised  loss function from cross-entropy to OHEM, the mean-IoU on eval dataset seems like really close, I dont know why this happen?

**Comment by WilhelmT** [comment]
Hello, thanks.

Could you clarify what you mean when you say that "the mean-IoU on eval dataset seems like really close"?

Are you using OHEM just for the supervised part or for both? When we used MSE loss for unlabeled images we had to alter the weighting paramater between the two losses (The details are in the paper). We have not experimented with using OHEM loss as far as I recall, so I'm not sure what hyperparamaters would be suitable in that case.
